README.md

```diff
@@ -99,7 +99,7 @@ This repo contains subgraphs defined using a set of standardized schemas. These
 | [CREAM Finance](https://cream.finance/) | 🛠 | 1.3.0 / 1.0.6 / 1.0.0 | [![CREAM Finance Arbitrum](./docs/images/chains/arbitrum.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-arbitrum) [![CREAM Finance Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-polygon) [![CREAM Finance BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-bsc) [![CREAM Finance Ethereum](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-ethereum)|
 | [dForce](https://dforce.network) | 🛠 | 1.3.0 / 1.0.1 / 1.0.0 | [![dforce Arbitrum](./docs/images/chains/arbitrum.png)]() [![dforce Avalanche](./docs/images/chains/avalanche.png)]() [![dforce BSC](./docs/images/chains/bsc.png)]() [![dforce Ethereum](./docs/images/chains/ethereum.png)]() [![dforce Optimism](./docs/images/chains/optimism.png)]() [![dforce Polygon](./docs/images/chains/matic.png)]() |
 | Euler Finance | 🔨 | | |
-| [Geist](https://geist.finance/markets) | 🛠 | 1.3.0 / 1.0.0 / 1.0.0 | [![Geist Protocl](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/geist-fantom) |
+| [Geist](https://geist.finance/markets) | 🛠 | 2.0.1 / 1.0.0 / 1.0.0 | [![Geist Protocl](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/geist-fantom) |
```

Comment:
No need to extend schema for Geist. Also no need for ARC and RWA

Reply:
Consider this a separate subgraph from `aave-v2`

Reply:
sounds good